security/backends/elasticsearch/writer.py

```diff
@@ -40,14 +41,24 @@
 MAX_VERSION = 9999
 
 
-def batch_delete_queryset(queryset, batch):
-    tmp_params = queryset._params.copy()
-    queryset._params['max_docs'] = batch
-    queryset._params['refresh'] = True
-    while True:
-        if queryset.delete()['deleted'] == 0:
-            break
-    queryset._params = tmp_params
+def batch_delete_queryset(queryset, batch, stdout, stderr, attempt=0):
+    try:
+        tmp_params = queryset._params.copy()
+        queryset._params['max_docs'] = batch
+        queryset._params['refresh'] = True
+        queryset._params['request_timeout'] = settings.ELASTICSERACH_CLEAN_LOGS_DELETE_REQUEST_TIMEOUT
+        while True:
+            stdout.write('.', ending='')
```

Comment:
delete is in batch. it's good to know something is happening

Reply:
![image](https://user-images.githubusercontent.com/424409/182929201-7859e777-5911-4d83-95ac-7af6f41110e3.png)